src/apps/convex/polygon/convex.deposit.token-fetcher.ts

```diff
@@ -0,0 +1,11 @@
+import { PositionTemplate } from '~app-toolkit/decorators/position-template.decorator';
+
+import { ConvexDepositSidechainTokenFetcher } from '../common/convex.deposit-sidechain.token-fetcher';
+
+@PositionTemplate()
+export class PolygonConvexDepositTokenFetcher extends ConvexDepositSidechainTokenFetcher {
+  groupLabel = 'Liqudity Pool Staking';
+  boosterAddress = '0xf403c135812408bfbe8713b5a23a04b3d48aae31';
+
+  isExcludedFromBalances = true;
```

Comment:
Had to otherwise balances were duplicated. Will have to do the same for LP farms on Arbitrum